docs/PUBLISHING.md

```diff
@@ -76,7 +85,7 @@ Make sure you set the following environment variables:
 Run the following command:
 
 ```sh
-make publish-bintray-debian RELEASE_TYPE=<production|snapshot>
+make publish-bintray-debian
```

Comment:
I *think* I tested it while updating the bintray deployment script, and removed the devel builds afterwards. In any case, I'm making a note to myself to make Travis CI upload Debian snapshot builds as well.